### DISCUSSION TITLE
Vector initialisation with some initial value

### DISCUSSION THREAD

**Discussion by sidforreal** [microsoft/vscode-cpptools#6951] [Q&A] [answered]
I am getting this error when I initialise a vector with some initial value:
<img width="944" alt="Screenshot 2021-02-11 at 3 07 45 AM" src="https://user-images.githubusercontent.com/32801674/107575924-78ff2700-6c16-11eb-960a-2860763de3ee.png">

#5643 was something related to this but in my case I am not using "C/C++ Clang command adaptor" I have only "Code Runner" along side "C/C++ for Visual Studio Code"

 `clang --version
`

> Apple clang version 12.0.0 (clang-1200.0.32.29)
> Target: x86_64-apple-darwin20.3.0
> Thread model: posix
> InstalledDir: /Library/Developer/CommandLineTools/usr/bin

**Comment by sean-mcmanus** [comment, marked as answer]
That error would occur if you're using C++03 instead of C++11 or newer. Can you check your cppStandard in c_cpp_properties.json or C_Cpp.default.cppStandard?

**Reply by sidforreal** [reply]
C_Cpp.default.cppStandard was set to empty by default. I changed it to c++17 and error is gone.
thanks a lot!!

**Reply by sean-mcmanus** [reply]
Yeah, that binary (Apple's version of clang) is reporting a default __cplusplus as 199711L, which is why it's the default. If you install brew clang on Mac, it will report a default __cplusplus of 201402L.
